user_docs/en/changes.t2t

```diff
@@ -287,7 +288,6 @@ eSpeak-NG, LibLouis braille translator, and Unicode CLDR have been updated.
 - Script for reporting the destination of a link now reports from the caret / focus position rather than the navigator object. (#14659)
 - Portable copy creation no longer requires that a drive letter be entered as part of the absolute path. (#14680)
 - If Windows is configured to display seconds in the system tray clock, using ``NVDA+f12`` to report the time now honors that setting. (#14742)
-- NVDA will now report unlabeled groupings that have useful position information, such as in recent versions of Microsoft Office 365 menus. (#14878)

```

Comment:
this is removing a line from the 2023.2 change log which should remain unchanged

Reply:
@michaelDCurran  It seems this PR has been  merged without taking this suggestion into account.

Reply:
Oh, totally missed this. thanks for calling this out.